### PR TITLE
Merged internal and public branch

### PR DESCRIPTION
**Pull request by nakwa**
Merge internal and public

### CODE REVIEW

File: schema/v1.0/sepa_credit_transfer.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
       "readOnly" : true
     },
     "fee_amount" : {
-      "$ref"     : "./base_types/base_types.json#definitions/fee",
```
Comment (schorsch) [inline schema/v1.0/sepa_credit_transfer.json]: think we hanged this to no allow negative, we could laos jsut kill the ref and put the definition inline

File: schema/v1.0/transaction.json [hunk 2]
```diff
@@ -12,7 +12,39 @@
       "readOnly" : true
     },
     "transaction_type" : {
-      "$ref"     : "./base_types/base_types.json#definitions/transaction_type",
```
Comment (schorsch) [inline schema/v1.0/transaction.json]: please also remove the ref-property if not needed anymore

File: schema/v1.0/transaction_type_details/credit_card_details.json [hunk 3]
```diff
@@ -29,10 +29,11 @@
       "readOnly"    : true
     },
     "cc_sequence" : {
-      "description" : "The CC sequence is the same for all related transactions",
```
Comment (schorsch) [inline schema/v1.0/transaction_type_details/credit_card_details.json]: the removed description was a bit better

**Comment by nakwa** [comment]
Those were changes that somehow got pushed to the public repository but not into internal master branch.
I had to merge it manually. I'll update it again, manually.

**Comment by schorsch** [comment]
thanks!

2015-06-29 16:14 GMT+02:00 REBOH <notifications@github.com>:

**Pull request merged**